src/components/DeleteTodo.js

```diff
@@ -0,0 +1,18 @@
+export default class DeleteTodo {
+  constructor ($todoList, loadTodo) {
+    this.loadTodo = loadTodo
+
+    $todoList.addEventListener('click', this.deleteTodo)
+  }
+
+  deleteTodo = ({ target }) => {
+    if (target.className === 'destroy') {
+      const todos = JSON.parse(localStorage.getItem('todos')) ?? []
+      const todo = todos.find(todo => todo.id === target.id)
+      todos.splice(todos.indexOf(todo), 1)
```

Comment:
```suggestion
      const todo = todos.filter(todo => todo.id !== target.id)
```

이렇게 `filter` 함수를 사용하면 바로 원하는 값만 제외하고 배열을 받을 수 있지않을까욥?! 
그러고 바로 `localStorage` 로 저장하는 방식으루요 🤔